tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
We should decide what we want our paradigm to be. I used the 0 value in listing timestamps to indicate "unlimited" because when requesting something that _must_ have a size, 0 is obviously non-sensical. I felt like it also fits in with Go's "zero" value idea. So if somebody created an int via `var x int`, then never assigned a more restricted value, the behaviour would be "get everything" rather than "get nothing".